azure-cognitiveservices-vision-face/tests/test_face.py

```diff
@@ -59,3 +60,54 @@ def test_face_detect(self):
         self.assertEqual(detected.face_attributes.age, 52.4)
         self.assertEqual(detected.face_attributes.gender, Gender.female)
         self.assertEqual(detected.face_attributes.emotion.happiness, 1.0)
+
+    def test_snapshot(self):
+        credentials = CognitiveServicesCredentials(
+            self.settings.CS_SUBSCRIPTION_KEY
+        )
+        face_client = FaceClient("https://westus2.api.cognitive.microsoft.com", credentials=credentials)
+
+        # Create a PersonGroup.
+        personGroupId = "69ff3e98-2de7-468e-beae-f78aa85200db"
+        newPersonGroupId = "fb644ecf-3ed0-4b25-9270-1d174b980afb"
+
+        face_client.person_group.create(personGroupId, "test", "test")
+
+        # Take a snapshot for the PersonGroup
+        apply_scope = ["Apply-Scope-Subscriptions"]
+        snapshot_type = "PersonGroup"
+
+        takeSnapshotResponse = face_client.snapshot.take(snapshot_type, personGroupId, apply_scope, raw=True)
+        takeOperationId = takeSnapshotResponse.headers["Operation-Location"].split("/")[2]
+
+        getOperationStatusResponse = face_client.snapshot.get_operation_status(takeOperationId)
+        operationStatus = getOperationStatusResponse.additional_properties["Status"]
+        
+        # Wait for take operation to complete.        
+        while operationStatus != "succeeded" and operationStatus != "failed":
+          getOperationStatusResponse = face_client.snapshot.get_operation_status(takeOperationId)
+          operationStatus = getOperationStatusResponse.additional_properties["Status"]
+          sleep(1)
+
+        self.assertEqual(operationStatus, "succeeded")
+
+        snapshotId = getOperationStatusResponse.additional_properties["ResourceLocation"].split("/")[2]
+        
+        # Apply the snapshot to a new PersonGroup.
+        applySnapshotResponse = face_client.snapshot.apply(snapshotId, newPersonGroupId, raw=True)
+        applyOperationId = applySnapshotResponse.headers["Operation-Location"].split("/")[2]
+
+        applyOperationStatusResponse = face_client.snapshot.get_operation_status(applyOperationId)
+        operationStatus = applyOperationStatusResponse.additional_properties["Status"]
+        
+        # Wait for apply operation to complete.
+        while operationStatus != "succeeded" and operationStatus != "failed":
+          applyOperationStatusResponse = face_client.snapshot.get_operation_status(applyOperationId)
+          operationStatus = applyOperationStatusResponse.additional_properties["Status"]
+          sleep(1)
```

Comment:
please change it to:
```python
if self.is_live:
    sleep(1)
```
since we want to sleep in real mode, but in record we can be fast. We will make our CI faster.

Reply:
Done. Thanks your review.